pal/system-info/web/system-info.ts

```diff
@@ -186,7 +186,7 @@ class SystemInfo extends EventTarget {
             [Feature.EVENT_ACCELEROMETER]: (window.DeviceMotionEvent !== undefined || window.DeviceOrientationEvent !== undefined),
             // @ts-expect-error undefined webkitGetGamepads
             [Feature.EVENT_GAMEPAD]: (navigator.getGamepads !== undefined || navigator.webkitGetGamepads !== undefined),
-            [Feature.EVENT_HANDLE]: this.isXR,
+            [Feature.EVENT_HANDLE]: EDITOR || PREVIEW,
```

Comment:
what if the web-mobilde and web-desktop platform, can they support EVENT_HANDLE ?
Or do we only support EVENT_HANDLE on EDITOR and PREVIEW for now ?

Reply:
EVENT_HANDLE, Only used for the xr 6dof handle. During the xr preview, the handle signal will be obtained from the device side and transferred to the web platform. It can only be used for development preview. The official web release does not support handle.